### DISCUSSION TITLE
Doc comment on Multiple vectors (/config-refs/schema/multi-vector)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1849] [Docs/blog comments]
# Doc comment on Multiple vectors (/config-refs/schema/multi-vector)

Syntax

https://weaviate.io/developers/weaviate/config-refs/schema/multi-vector

**Comment by lucaspeyrin** [comment]
But is it possible to create a schema with multiple vectors using the REST Api ?

I tried transforming the Python version into an API call, but with multiple "vectorizer_config":[] it doesn't seem to work.

Would love to know if it's possible !
